wagtail/tests/modeladmintest/models.py

```diff
@@ -124,3 +124,8 @@ class Friend(models.Model):
 
     def __str__(self):
         return self.first_name
+
+
+class PrepopulatedFieldModel(models.Model):
+    name = models.CharField(max_length=100)
+    slug = models.SlugField()
```

Comment:
If at all possible, I'd like to avoid adding another model to the test suite to specifically test this feature. Technically, there's no reason why the target field needs to be a `SlugField`, is there? Maybe we could just take an existing model (like `Book`) and add `prepopulated_fields` to an existing `ModelAdmin` class, with any old field being set from some others?